fitbenchmarking/fitting/minimizers.json

```diff
@@ -1,8 +1,8 @@
 {
-    "mantid" : ["BFGS", "Conjugate gradient (Fletcher-Reeves imp.)",
-                "Conjugate gradient (Polak-Ribiere imp.)",
-                "Damped GaussNewton",
-                "Levenberg-Marquardt", "Levenberg-MarquardtMD",
-                "Simplex","SteepestDescent",
-                "Trust Region"]
+    "mantidv3.12" : ["BFGS", "Conjugate gradient (Fletcher-Reeves imp.)",
```

Comment:
we are now on 3.13 of mantid

Reply:
This is not currently used but a start to expanding mantid to other minimizers/fitting algorithms. Mantid 3.13 will be added too, but currently those are the minimizers utilized by Mantid 3.12 .